build/lender/Lender.sol

```diff
@@ -454,6 +450,14 @@ contract Lender {
     return feenominator > 0 ? a / feenominator : 0;
   }
 
+  /// @notice retrieves the zc token for the given market
+  /// @param u address of the underlying
+  /// @param m uint256 representing the maturity of the market
+  /// @return address of the zc token for the market
+  function getZcToken(address u, uint256 m) internal returns (address) {
```

Comment:
wasn't this just `zcToken` ? i generally view `get` as a garbage word.

Reply:
No, I created this helper function to deal with the stack problem. But just `zcToken` is much cleaner.